docs/topics_5/web-updating-an-analysis.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='updating_analysis_{context}']
+= Updating the analysis configuration
+
+You might want to to make some changes in a project that you saved, perhaps to test an application against a different transformation target, add a new custom rule, or change an advanced setting. The *Analysis configuration* screen of the {ProductShortName} {WebName} makes it easy to make these types of changes and run a new analysis of the project without having to create and define it from the beginning.
```

Comment:
This is a bit turgid. Suggest this:

You can update an analysis configuration, for example, with a different transformation target, advanced option, or a custom rule. Then you can run the updated analysis in your project.